language-design.md

```diff
@@ -17,7 +17,41 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+Quote 1: "Why shouldn’t those 'interfaces' be as humanely designed as the ones we tap and swipe?" [Pavlus, 2012].
+
+Here Pavlus asks why programming languages' interfaces are not as "humanely
+designed" as software's graphical interfaces are. This quote stood out to us
+because Pavlus is trying to make a point, but actually ends up asking a
+question that is very easy to answer! One answer is that in general, software
+(programming languages included) is designed towards its users. The software
+Pavlus is referring to is probably targeted at a much broader audience, so it
+needs to be easier. Also, we take issue with Pavlus implying that programming
+is comparable to tapping and swiping. Is Pavlus claiming that programming
+should be as easy as Angry Birds? I doubt that would be a productive language.
```

Comment:
Your phrasing makes it seem like you're saying that the ease of use of something in inversely proportional to its usefulness, but that's pretty obviously false -- in fact, you could easily make a claim of exactly the opposite. It would be nice if you explained your opinion a bit more